src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -1772,8 +1780,11 @@ public async Task DispatchCompletion_DispatchesLingerAfterDisconnect()
     }
 
     [Fact]
-    public async Task MultipleSyncMethodsExecuteConcurrentlyOnServer()
+    public async Task MultipleSyncMethodsExecuteConcurrentlyOnServer_WithClearedSyncContext()
```

Comment:
nit: Test for the default non concurrent behavior.

Reply:
Good idea. I'll add one.